scipy/integrate/_quadpack_py.py

```diff
@@ -661,17 +678,31 @@ def tplquad(func, a, b, gfun, hfun, qfun, rfun, args=(), epsabs=1.49e-8,
     --------
     Compute the triple integral of ``x * y * z``, over ``x`` ranging
     from 1 to 2, ``y`` ranging from 2 to 3, ``z`` ranging from 0 to 1.
+    That is, :math:`\\int^{x=2}_{x=1} \\int^{y=3}_{y=2} \\int^{z=1}_{z=0} x y z dz dy dx`.
 
     >>> from scipy import integrate
     >>> f = lambda z, y, x: x*y*z
-    >>> integrate.tplquad(f, 1, 2, lambda x: 2, lambda x: 3,
-    ...                   lambda x, y: 0, lambda x, y: 1)
-    (1.8750000000000002, 3.324644794257407e-14)
+    >>> integrate.tplquad(f, 1, 2, 2, 3, 0, 1)
+    (1.8749999999999998, 3.3246447942574074e-14)
+
+    Calculate :math:`\\int^{x=1}_{x=0} \\int^{y=1-2x}_{y=0} \\int^{z=1-x-2y}_{z=0} x y z dz dy dx`.
+    Note: qfun/rfun takes arguments in the order (x, y), even though f takes arguments in the order (z, y, x).
```

Comment:
To reference function or parameters, use single backticks. Otherwise double backticks.
```suggestion
    Note: `qfun`/`rfun` takes arguments in the order (x, y), even though ``f`` takes arguments in the order (z, y, x).
```